src/index.js

```diff
@@ -9,6 +9,7 @@ import BrowserProxy from './browser-proxy';
 
 const BUILD_ID     = process.env['BROWSERSTACK_BUILD_ID'];
 const PROJECT_NAME = process.env['BROWSERSTACK_PROJECT_NAME'];
+const BROWSERSTACK_RESOLUTION = process.env['BROWSERSTACK_RESOLUTION'] || '1024x768';
```

Comment:
Please, don't enforce default resolution, leave it to BrowserStack. Also, I think it's better to call it `BROWSERSTACK_DISPLAY_RESOLUTION`.